tests/test_utils.py

```diff
@@ -40,6 +39,18 @@ def test_get_image_location_ok(tmpdir):
     )
 
 
+def test_get_image_location_unicode_path_ok(tmpdir):
+    image = "ąþð/→µŋ.png"
+    path = six.text_type(tmpdir.mkdir('ąþð').join("→µŋ.png"))
+    image_list = [path]
+
+    assert path == plotextractor.output_utils.get_image_location(
+        image,
+        six.text_type(tmpdir),
```

Comment:
why we need to do ``six`` here?

Reply:
We don't need it anymore at all, but it's also not harmful at all. Please notice that it's in many places and I didn't want to loose time on removing it. Of course I can do it if you wish.